### ISSUE TITLE
Trezor T BOM and schematic mismatch

### ISSUE THREAD

**Issue by micmys** [trezor/trezor-firmware#1459]
I noticed there are differences in the BOM for Trezor model T:
[https://github.com/trezor/trezor-firmware/blob/master/docs/hardware/model-t/assets/bom.txt](https://github.com/trezor/trezor-firmware/blob/master/docs/hardware/model-t/assets/bom.txt)
and its schematic:
[https://github.com/trezor/trezor-firmware/blob/master/docs/hardware/model-t/assets/schematic.png](https://github.com/trezor/trezor-firmware/blob/master/docs/hardware/model-t/assets/schematic.png)

For example:
- transistors T1/T2 with their accompanying components are absent in the BOM
- IC1 is also nowhere to be found
- resistors R9, R10 have different values in both documents
- (probably some more discrepancies)

I assume that the schematic is newer & correct and BOM should be adjusted, but this probably needs clarification.

**Comment by prusnak** [comment]
Right, we need to clean that up. In the meantime you can check the Eagle files here: https://github.com/trezor/trezor-hardware/tree/master/electronics/trezor_model_t
